src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -111,7 +114,8 @@ public object VisitSqlRoot(SqlRootExpression expression, SearchOptions context)
                     StringBuilder.Append("CAST(0 AS bit) AS IsRawResourceMetaSet, ");
                 }
 
-                StringBuilder.AppendLine(VLatest.Resource.RawResource, resourceTableAlias);
+                StringBuilder.Append(VLatest.Resource.RawResource, resourceTableAlias);
+                StringBuilder.AppendLine((searchParamInfo == null || searchParamInfo.Name == KnownQueryParameterNames.LastUpdated) ? string.Empty : $", {TableExpressionName(_tableExpressionCounter)}.SortValue");
```

Comment:
Elsewhere in the SQL codegen, we try to minimize string allocations by chaining `Append()` calls instead of using string interpolation